docs/docs/how-tos/nebari-azure.md

```diff
@@ -28,7 +28,7 @@ Billing for Azure services is done on a per-subscription basis. For a list of th
 [Microsoft Azure Offer Details](https://azure.microsoft.com/support/legal/offer-details/) and
 [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/ask/intro-kubernetes) for a quick overview of the Kubernetes services.
 
-For a more detailed cost estimate, please also refer to our \[Conceptual guides\] for more information regarding the basic infrastructure provided by Nebari.
+For a more detailed cost estimate, please also refer to our [Conceptual guides](/explanations/) for more information regarding the basic infrastructure provided by Nebari.
```

Comment:
```suggestion
Refer to our [Conceptual guides](/docs/explanations/) for more information regarding the basic infrastructure provided by Nebari.
```
We don't have much information regarding costs right now (in the works), so I removed the first sentence, also fixed the link